pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/TransformContext.java

```diff
@@ -48,6 +48,7 @@ public class TransformContext {
   private String key;
   private Map<String, String> properties;
   private String outputTopic;
+  private boolean dropCurrentRecord;
```

Comment:
What if we set the value to null instead of adding a field ?

Reply:
In KV schemas for example, Value is allowed to be null. We use this in CDC to send a mutation to indicate that a record was deleted (so value would be null, but still a record would be sent). 

I thought about setting both key and value to null to indicate the delete, but found the flag more explicit.

Reply:
We should have some tests with KV with null key or value.